src/index.ts

```diff
@@ -1,10 +1,11 @@
 import OAuth from './OAuth';
 import RestClient from './RestClient';
+import * as types from './types';
 
-const BigCommerce = {
-  Auth: OAuth,
-  Rest: RestClient,
-};
+class BigCommerce {
+  static Auth = OAuth;
+  static Rest = RestClient;
+  static Types = types;
```

Comment:
🍹 This might be outside the scope of this ticket - but I can't remember our original reason for exporting type definitions in general. The REST/OAuth methods are already typed when they're imported by consumers of the client, and we're not showing any demonstration of using the exported types in the README. I might suggest talking to the team about whether or not we actually want to explicitly export these.

Reply:
Updated this back to an object

Reply:
We're still exporting types here - is there any good reason we should do this?

Reply:
Updated!